chia/_tests/util/test_testnet_overrides.py

```diff
@@ -27,26 +13,6 @@ def test_testnet11() -> None:
     }
 
 
-def test_testnet10_existing() -> None:
-    overrides: Dict[str, Any] = {
-        "HARD_FORK_HEIGHT": 42,
-        "HARD_FORK_FIX_HEIGHT": 3426000,
-        "PLOT_FILTER_128_HEIGHT": 42,
-        "PLOT_FILTER_64_HEIGHT": 42,
-        "PLOT_FILTER_32_HEIGHT": 42,
-    }
-    update_testnet_overrides("testnet10", overrides)
-    assert overrides == {
-        "SOFT_FORK2_HEIGHT": 3000000,
-        "SOFT_FORK4_HEIGHT": 4465000,
-        "HARD_FORK_HEIGHT": 42,
-        "HARD_FORK_FIX_HEIGHT": 3426000,
-        "PLOT_FILTER_128_HEIGHT": 42,
-        "PLOT_FILTER_64_HEIGHT": 42,
-        "PLOT_FILTER_32_HEIGHT": 42,
-    }
-
-
```

Comment:
Looks like we are missing a `test_testnet11_existing()` test. Could be addressed here or in another pr.